demo.tsx

```diff
@@ -21,6 +21,7 @@ interface FormData {
 
 export default function Demo() {
     const onSubmit: FormSubmitHandler<FormData> = (values, fc) => {
+		// eslint-disable-next-line no-console
         console.log(values)
```

Comment:
a complete nitpick, but I usually try to keep the eslint disable  on the same line so they can both be removed at once:
`console.log(values) // eslint-disable-line no-console`